javascript/chronometer.js

```diff
@@ -1,34 +1,46 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+    }, 1000);
+    if(callback){
+      setInterval((callback), 1000);
+    }
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60);
   }
 
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime - this.getMinutes(this.currentTime)*60;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value < 10 ){
+      return "0"+value.toString();
+    } else {
+      return value.toString().slice(0, 2)
+    }
   }
 
   stop() {
-    // ... your code goes here
+    return clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    let formattedMinutes = this.computeTwoDigitNumber(this.getMinutes(this.currentTime));
+    let formattedSeconds = this.computeTwoDigitNumber(this.getSeconds(this.currentTime));
```

Comment:
If you look back at getMinutes() method above - it does not expect any arguments, so you do not want to be passing anything when you call it ;)